include/megaapi.h

```diff
@@ -10953,6 +10966,15 @@ class MegaApi
          */
         void setNodeFavourite(MegaNode *node, bool fav, MegaRequestListener *listener = NULL);
 
+        /**
+         * @brief Get a list of favourite nodes.
+         *
```

Comment:
The information about the type of request and the valid members is missing. See others for reference, please.